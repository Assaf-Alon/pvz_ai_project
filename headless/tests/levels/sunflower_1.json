{
    "50": [["normal", 3]]
}
