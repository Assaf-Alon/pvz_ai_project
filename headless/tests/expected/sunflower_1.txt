[1] Planted Sunflower in (0, 0). Total: 0.
[51] Sun generated by Sunflower in (0, 0). Total: 25.
[101] Sun generated by Sunflower in (0, 0). Total: 50.
[101] Sun autospawned. Total: 75.
[151] Sun generated by Sunflower in (0, 0). Total: 100.
[201] Sun generated by Sunflower in (0, 0). Total: 125.
[202] Sun autospawned. Total: 150.
[251] Sun generated by Sunflower in (0, 0). Total: 175.
[301] Sun generated by Sunflower in (0, 0). Total: 200.
[303] Sun autospawned. Total: 225.
[351] Sun generated by Sunflower in (0, 0). Total: 250.
[401] Sun generated by Sunflower in (0, 0). Total: 275.
[404] Sun autospawned. Total: 300.
[451] Sun generated by Sunflower in (0, 0). Total: 325.
[500] Zombie being spawned at lane 3
[501] Sun generated by Sunflower in (0, 0). Total: 350.
[505] Sun autospawned. Total: 375.
[547] Zombie in (3, 9) moved to (3, 8).
[551] Sun generated by Sunflower in (0, 0). Total: 400.
[594] Zombie in (3, 8) moved to (3, 7).
[601] Sun generated by Sunflower in (0, 0). Total: 425.
[606] Sun autospawned. Total: 450.
[641] Zombie in (3, 7) moved to (3, 6).
[651] Sun generated by Sunflower in (0, 0). Total: 475.
[688] Zombie in (3, 6) moved to (3, 5).
[701] Sun generated by Sunflower in (0, 0). Total: 500.
[707] Sun autospawned. Total: 525.
[735] Zombie in (3, 5) moved to (3, 4).
[751] Sun generated by Sunflower in (0, 0). Total: 550.
[782] Zombie in (3, 4) moved to (3, 3).
[801] Sun generated by Sunflower in (0, 0). Total: 575.
[808] Sun autospawned. Total: 600.
[829] Zombie in (3, 3) moved to (3, 2).
[851] Sun generated by Sunflower in (0, 0). Total: 625.
[876] Zombie in (3, 2) moved to (3, 1).
[901] Sun generated by Sunflower in (0, 0). Total: 650.
[909] Sun autospawned. Total: 675.
[923] Zombie in (3, 1) moved to (3, 0).
[951] Sun generated by Sunflower in (0, 0). Total: 700.
[970] Zombie in (3, 0) attempted entering the house.
[970] Zombie in (3, 0) triggered a lawnmower.
[970] Lawnmower in (3, 0) attacked.
[970] Zombie in (3, 0) was damaged. HP: -9818.
[970] Zombie in (3, 0) was killed.
[970] You've won!
